test/verify/check-superuser

```diff
@@ -341,6 +345,23 @@ session include system-auth
         b.wait_not_present(".pf-c-modal-box:contains('Switch to administrative access')")
         b.wait_not_present(".pf-c-alert:contains('Web console is running in limited access mode.')")
 
+    @unittest.skipIf(DEFAULT_IMAGE != "fedora-coreos", "Slow test, runs only on fedora-coreos")
```

Comment:
Yikes.  Can you take a look at this @martinpitt ?  Is this the best way?

Reply:
It could move into an existing test with a more standard `if m.image == ["fedora-coreos"]:` check, which would also save a VM start/boot.

Reply:
Actually, we want to test this with the python bridge, no? So fedora-coreos is the wrong image to run this on.